src/app/server/Server.h

```diff
@@ -207,6 +207,8 @@ struct ServerInitParams
     // data model it wants to use. Backwards-compatibility can use `CodegenDataModelProviderInstance`
     // for ember/zap-generated models.
     chip::app::DataModel::Provider * dataModelProvider = nullptr;
+
+    bool advertiseCommissionable = CHIP_DEVICE_CONFIG_ENABLE_PAIRING_AUTOSTART;
```

Comment:
```suggestion
    bool advertiseCommissionableIfNoFabrics = CHIP_DEVICE_CONFIG_ENABLE_PAIRING_AUTOSTART;
```
is what it really is, right?

Reply:
yeah, i like that - thanks. Updated in a separate commit because it has to be changed in a couple of places.